nist/int.go

```diff
@@ -37,8 +39,9 @@ var two = big.NewInt(2)
 // whose target is assumed never to change.
 //
 type Int struct {
-	V big.Int  // Integer value from 0 through M-1
-	M *big.Int // Modulus for finite field arithmetic
+	V          big.Int          // Integer value from 0 through M-1
+	M          *big.Int         // Modulus for finite field arithmetic
+	endianness binary.ByteOrder // Endianness considered for this int
```

Comment:
All right for the public. I also hesitated creating our own type but then I chose the *standard* way. But since as you noted it might break the thing, I now think that's also best to declare our own...